### PR TITLE
#170947538 setup unit testing

### PR DESCRIPTION
**Pull request by Ntare22**
**What does this PR do?**
* Setup unit testing. 

**Description of Task to be completed?**
* install express
* install chai, mocha, body-parser, @babel, nodemon, and nyc
* setup babel for the project
* start server in index.js file
* made a test directory
* write a sample test

**How should this be manually tested?**
* clone repo
* enter the project
* run 'npm install' in the terminal
* run 'npm test' in the terminal

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [#170947538](https://www.pivotaltracker.com/story/show/170947538)

**Screenshots (if appropriate)**
image

### CODE REVIEW

**Review by bbaime98 — COMMENTED** [review]
since our tests will need a database, You can add a sequelize test script too which clears the database and migrate.

**Comment by Ntare22** [comment]
I would have to install sequelize and set it up so that I can test it.... I'll work on it once we merge our work and I'm able to get use sequelize!

**Comment by bbaime98** [comment]
> I would have to install sequelize and set it up so that I can test it.... I'll work on it once we merge our work and I'm able to get use sequelize!

Okay. That's understandable.

**Review by bbaime98 — APPROVED** [review]

**Review by StevenDerrick — APPROVED** [review]

**Review by Baraka-Mugisha — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,16 @@
...
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
 
-const isProduction = process.env.NODE_ENV === "production";
```
Comment (Mnickii) [inline src/index.js:13]: why did you remove the production configurations? We will need them when setting up a prod environment.

**Review by Mnickii — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (Ntare22) [reply src/index.js:13]: I just thought I'll put it there once I create the pipelines? Can i put it back up and push it?
